tez-dist/src/main/assembly/tez-dist-minimal.xml

```diff
@@ -26,6 +26,7 @@
         <exclude>org.apache.tez:tez-aux-services</exclude>
         <exclude>org.apache.tez:tez-ext-service-tests</exclude>
         <exclude>org.apache.tez:tez-ui</exclude>
+        <exclude>org.apache.tez:tez-docs</exclude>
```

Comment:
For some reason, the latest Maven Assembly plugin dies (NPE) when trying to process this module.  I don't think it's required in the minimal distribution anyway.